xendit-android/src/main/java/com/xendit/Models/CardMetadata.java

```diff
@@ -0,0 +1,40 @@
+package com.xendit.Models;
+
+import com.google.gson.annotations.SerializedName;
+
+public class CardMetadata {
+    @SerializedName("bank")
+    private String bank;
+
+    @SerializedName("country_code")
+    private String country_code;
+
+    @SerializedName("type")
+    private String type;
+
+    @SerializedName("brand")
+    private String brand;
+
+    @SerializedName("card_art_url")
+    private String card_art_url;
+
+    public String getBank() {
+        return bank;
+    }
+
+    public String getCountry_code() {
```

Comment:
i think we should decide in the future if we want to do `getCountry_code` or `getCountryCode`. i think currently still mix of both style haha

Reply:
let me change this. I auto-generated this code and didnt check what came out.